CMakeLists.txt

```diff
@@ -0,0 +1,148 @@
+# We require 3.9 because of the use of $<TARGET_OBJECTS:crt1>
+cmake_minimum_required(VERSION 3.9)
+project(MUSL)
+enable_language(C ASM)
+include(CheckCCompilerFlag)
+
+if(WASM)
```

Comment:
How does `if(WASM)` work? Does cmake know about wasm? Or is it passed on the command-line?

Reply:
I'm adding it  to the cmake toolchain file on the waterfall.  But you can also do `-DWASM=1` on the command line too.

Reply:
Maybe make it into a [cmake option](https://cmake.org/cmake/help/v3.0/command/option.html) then.